docs/actions.md

```diff
@@ -538,19 +538,32 @@ Naming conventions:
 
 Creating the zip file:
 
-1. Name your main Python file `__main__.py`
-2. A file named `requirements.txt` contains the pip modules and versions to install
-3. Run the following docker command to create the virtualenv directory: 
-
+1. A file named `requirements.txt` contains the pip modules and versions to install
+2. Run the following docker command to create the virtualenv directory: 
+    * Python 2
+ ```
+ $ docker run -v "$PWD:/tmp" openwhisk/python2action sh -c "cd tmp; virtualenv virtualenv; source virtualenv/bin/activate; pip install -r requirements.txt;"
+ ```
+    * Python 3
  ```
- $ docker run -v "$PWD:/tmp" openwhisk/pythonaction sh -c "cd tmp; virtualenv virtualenv; source virtualenv/bin/activate; pip install -r requirements.txt;"
+ $ docker run -v "$PWD:/tmp" openwhisk/python3action sh -c "cd tmp; virtualenv virtualenv; source virtualenv/bin/activate; pip install -r requirements.txt;"
```

Comment:
Add `--rm` to the command so the user doesn't manually need to remove the container afterwards.